pubspec.yaml

```diff
@@ -14,6 +14,7 @@ environment:
 dependencies:
   args: ^2.3.0
   build_cli_annotations: ^2.0.0
+  collection: ^1.16.0
```

Comment:
are we depending on that somewhere in this PR still? Or this was before some refactorings and is not needed?

Reply:
Feel free resolve if it is used.